File: DSA_In_Javascript/03_Mathematics/02_Palindrome_Number.js
const palindromeNumber = function(num) {
  let revNum = 0, temp = num
  while (temp > 0) {
    revNum = (revNum * 10) + temp % 10
    temp = Math.floor(temp / 10)
  }
  return revNum === num
}

console.log(palindromeNumber(787))
console.log(palindromeNumber(78007))
console.log(palindromeNumber(780087))

File: DSA_In_CSharp/03_Mathematics/02_Palindrome_Number.cs
namespace Mathematics {
    public class PalindromeNumber {
        public bool checkIsPalindromeNumber(int num) {
            int temp = num, revNum = 0;
            while (temp > 0) {
                revNum *= 10;
                revNum = revNum + (temp % 10);
                temp /= 10;
            }
            return num == revNum;
        }
        public void Test() {
            Console.WriteLine("101 : "+ checkIsPalindromeNumber(101));
            Console.WriteLine("1011 : "+ checkIsPalindromeNumber(1011));
        }
    }
}